packages/launchpad/src/migration/OptOutModal.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <StandardModal
+    :model-value="true"
+    :title="t('migration.renameAuto.modal.title')"
+    @update:modelValue="emit('cancel')"
+  >
+    <Alert
+      status="warning"
+      :title="t('migration.renameAuto.modal.warning')"
+      :icon="WarningIcon"
+      icon-classes="icon-dark-orange-400"
+    />
+    <MigrationTitle
+      :title="t('migration.renameAuto.title')"
+      class="mt-24px"
+    />
+    <MigrationList>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line1') }}
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line2') }}
+        <CodeTag class="text-jade-500">
+          [filename].cy.[ext]
+        </CodeTag>
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line3') }}
+      </MigrationListItem>
+      <MigrationListItem>
+        {{ t('migration.renameAuto.modal.line4') }}
+        <CodeTag class="text-jade-500">
+          [filename].cy.[ext]
+        </CodeTag>
+      </MigrationListItem>
+      <MigrationListItem>
+        <i18n-t
+          scope="global"
+          keypath="migration.renameAuto.modal.line5"
+        >
+          <CodeTag class="text-jade-500">
+            [filename].cy.[ext]
+          </CodeTag>
+        </i18n-t>
+      </MigrationListItem>
+      <MigrationListItem>
+        <i18n-t
+          scope="global"
+          keypath="migration.renameAuto.modal.line6"
+        >
+          <CodeTag class="text-jade-500">
+            [filename].cy.[ext]
+          </CodeTag>
+        </i18n-t>
+      </MigrationListItem>
```

Comment:
If you wanted to get mega-fancy making this dynamic, you can do:

```
      <MigrationListItem
        v-for="(listItem, index) in listItems.length"
        :key="index"
      >
        <i18n-t
          scope="global"
          :keypath="`migration.renameAuto.modal.listItems[${index}]`"
        >
          <CodeTag class="text-jade-500">
            [filename].cy.[ext]
          </CodeTag>
        </i18n-t>
      </MigrationListItem>
```

Where the script tag has:
```
const { t, messages, locale } = useI18n()
const { listItems } = messages.value[locale.value].migration.renameAuto.modal.listItems
```

and `en-US` has:

```
      "modal": {
        "title": "Change the existing spec file extension",
        "warning": "We recommend using the default extension to avoid inconsistencies, framework conflicts, and confusion with your team.",
        "listItems": [
          "Cypress now supports the ability to create new spec files from within the UI for both E2E and component specs.",
          "All new spec files created within Cypress will use the default pattern of: {0}",
          "We want to rename your existing specs so that they have a consistent filename pattern for both E2E and component testing.",
          "All documentation and example code will be using: {0}",
          "We've changed the placement of component specs to be next to their source files (e.g. src/Button.jsx and src/Button.cy.jsx)",
          "The new default pattern of {0} prevents targeting conflicts with other testing frameworks. (e.g. Jest)"
        ],
```

Reply:
That feels like we're beginning to make assumptions about the text contents in the code. Sure, right now they all only use `[filename].cy.[ext]` - but if we add another bullet point that needs a different code tag, we'd have to start refactoring the component.

The duplication here (6 `MigrationListItem`) isn't so bad that I feel the need to refactor it yet. If we had a dozen I'd agree entirely.

Reply:
All good in the hood! Just want to clarify one thang

> if we add another bullet point that needs a different code tag, we'd have to start refactoring the component.

If we add another bullet point, we wouldn't have to refactor -- the original one would be accessed as `{0}` and the additional one would be accessed as `{1}`

Reply:
Sure. We'd just then be cross-referencing via magic numbers, and using/ignoring different interpolated values in different strings, which is a bad sign all around.